src/Organizations/OrganizationDetails/OrganizationDetailsContainer.js

```diff
@@ -164,6 +167,10 @@ OrganizationDetailsContainer.manifest = Object.freeze({
     ...categoriesResource,
     accumulate: true,
   },
+  organizationTypes: {
```

Comment:
You do not need to fetch all types, just fetch current organization types only. Also we try to avoid mutators usage, the common aproach is to write hooks with react-query instead.

Reply:
I fetch the organizatonTypes now with the Ids saved in the organization. But therefor I need to call the GET at a time were the Ids of the organization.organizationTypes are available. 
I does not look as the best solution for me, but for outsourcing the GET in a separate useEffect or react-query-hook I just don't have access to the necessary Ids anymore. 
@AndreiShumski2   If you have a solution for solving that scope problem, please let me know.

Reply:
You can create a separate hook ex. `useOrganizationTypes` and pass current org type ids. And make GET request when organization is fetched. The same approach as for org integrations - [see](https://github.com/folio-org/ui-organizations/blob/master/src/Organizations/OrganizationDetails/OrganizationDetailsContainer.js#L44)

Reply:
Thanks for the hint @AndreiShumski2 !
Can you please re-review?

Reply:
Does it make sense to mix the technologies (mutators and  react-query) within a class? 
Maybe an option would be to create a succession ticket where all occurrences of mutators in the class are replaced?

Reply:
as you can see it's not a class, all new functionality is implemented with react-query, stripes-connect mutators will be migrated to queries in future